hera_pspec/pspecdata.py

```diff
@@ -1448,6 +1516,51 @@ def get_Q_alt(self, mode, allow_fft=True):
         Q_alt = np.einsum('i,j', m.conj(), m) # dot it with its conjugate
         return Q_alt
 
+    def get_Q(self, mode, feed_pol=False):
+        '''
+        Computes Q_alpha(i,j), which is the response of the data covariance to the bandpower (dC/dP_alpha). 
+        This includes contributions from primary beam.
+        
+        Parameters
+        ----------
+        mode : int
+            Central wavenumber (index) of the bandpower, p_alpha.
+
+        feed_pol : str/int/bool, optional
```

Comment:
I'm a little confused about what happens here if `feed_pol = False`. In `PSpecBeam.beam_normalized_response()`, it looks as if passing `pol=False` will result in a `ValueError`. I don't think the tests cover the default case (where `feed_pol=False`) either, so might be worth checking this has the expected behaviour.

Reply:
Also, why is this argument called `feed_pol`, instead of just `pol` (as in most other methods)?

Reply:
If polarization is not found by `PSpecBeam.beam_normalized_response()`, it raises the `ValueError: Do not have the right polarization information`. This test is rather covered in the `test_pspecbeam` by passing a different polarization. 

`feed_pol` was chosen to avoid any potential conflicts with `pol` existing in the code. Closely checking, it seemed it should be fine, so have replaced it now.